src/main/java/core/basesyntax/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationsStrategy;
+
+public class BalanceOperation implements OperationsStrategy {
```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationsStrategy {
```
please rename the same way all other handlers